minifi-toolkit/minifi-toolkit-assembly/NOTICE

```diff
@@ -0,0 +1,16 @@
+Apache NiFi
+Copyright 2014-2016 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+===========================================
+Apache Software License v2
+===========================================
+
+The following binary components are provided under the Apache Software License v2
+
+  (ASLv2) Apache NiFi
```

Comment:
Apache NiFi's notice includes more than just this. It should be this[1]:
> 
> Apache NiFi
> Copyright 2014-2016 The Apache Software Foundation
> 
> This product includes software developed at
> The Apache Software Foundation (http://www.apache.org/).
> 
> This product includes the following work from the Apache Hadoop project:
> 
> BoundedByteArrayOutputStream.java adapted to SoftLimitBoundedByteArrayOutputStream.java

[1] https://github.com/apache/nifi/blob/nifi-0.6.0/NOTICE